components/steps/w-step.vue

```diff
@@ -54,45 +54,42 @@ const getAriaLabel = (props) => {
 const stepClasses = computed(() => [
   ccStep.step,
   {
-    [ccStep.stepVertical]: vertical.value,
-    [ccStep.stepVerticalLeft]: vertical.value && left.value,
-    [ccStep.stepVerticalRight]: vertical.value && !left.value,
-    [ccStep.stepHorizontal]: !vertical.value,
+    [ccStep.vertical]: vertical.value,
+    [ccStep.alignLeft]: vertical.value && left.value,
+    [ccStep.alignRight]: vertical.value && !left.value,
+    [ccStep.horizontal]: !vertical.value,
   },
 ]);
 
-const horizontalClasses = computed(() => [
+const lineHorizontalClasses = computed(() => [
   ccStep.stepLine,
-  ccStep.stepLineHorizontalLeft,
+  ccStep.lineHorizontalAlignLeft,
   {
-    [ccStep.stepLineHorizontal]: !vertical.value,
-    [ccStep.stepLineIncomplete]: !props.active && !props.complete,
-    [ccStep.stepLineComplete]: props.active || props.complete,
+    [ccStep.lineHorizontal]: !vertical.value,
+    [ccStep.lineIncomplete]: !props.active && !props.complete,
+    [ccStep.lineComplete]: props.active || props.complete,
   },
 ]);
 
-const stepDotClasses = computed(() => [
-  ccStep.stepDot,
+const dotClasses = computed(() => [
+  ccStep.dot,
   {
-    [ccStep.stepDotVertical]: vertical.value,
-    [ccStep.stepDotVerticalLeft]: vertical.value && left.value,
-    [ccStep.stepDotVerticalRight]: vertical.value && !left.value,
-    [ccStep.stepDotHorizontal]: !vertical.value,
-    [ccStep.stepDotIncomplete]: !(props.active || props.complete),
-    [ccStep.stepDotActive]: props.active || props.complete,
+    [ccStep.dotAlignRight]: vertical.value && !left.value,
+    [ccStep.dotHorizontal]: !vertical.value,
+    [ccStep.dotIncomplete]: !(props.active || props.complete),
+    [ccStep.dotActive]: props.active || props.complete,
   },
 ]);
 
-const stepLineClasses = computed(() => [
-  ccStep.stepLine,
-  ccStep.stepLineHorizontalRight,
+const lineClasses = computed(() => [
+  ccStep.line,
+  ccStep.lineHorizontalAlignRight,
   {
-    [ccStep.stepLineVertical]: vertical.value,
-    [ccStep.stepLineVerticalLeft]: vertical.value && left.value,
-    [ccStep.stepLineVerticalRight]: vertical.value && !left.value,
-    [ccStep.stepLineHorizontal]: !vertical.value,
-    [ccStep.stepLineIncomplete]: !props.complete,
-    [ccStep.stepLineComplete]: props.complete,
+    [ccStep.lineVertical]: vertical.value,
+    [ccStep.lineAlignRight]: vertical.value && !left.value,
+    [ccStep.lineHorizontal]: !vertical.value,
+    [ccStep.lineIncomplete]: !props.complete,
+    [ccStep.lineComplete]: props.complete,
```

Comment:
Maybe move these ones up to the array level, since they are always mutually exclusive? 🙂 
```
    vertical.value ? ccStep.lineVertical : ccStep.lineHorizontal,
    props.complete ? ccStep.lineComplete : ccStep.lineIncomplete,
```
(...and there are a few more places where this might be better? Wdyt? 😄)

Reply:
yes! That would be nicer! I'll update it! :)